### PR TITLE
Start the glossary

### PR DESCRIPTION
**Pull request by badboy**
[doc only]

### CODE REVIEW

File: docs/appendix/glossary.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+They are defined in [metrics.yaml](https://mozilla.github.io/glean_parser/metrics-yaml.html) files.
+
+Glean itself provides some metrics out of the box.
+
+## Ping
+
+A ping is an entity used to bundle related metrics.
+The Glean SDK provides default pings and allows for custom ping, see [Glean Pings](../user/pings/index.md).
+
+## Pipeline
```
Comment (brizental) [inline docs/appendix/glossary.md]: Uh, nice addition :)

File: docs/appendix/glossary.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Glean SDK book
+
+This documentation.
+
+## Metric
+
+Metrics are the individual things being measured using Glean.
+They are defined in [metrics.yaml](https://mozilla.github.io/glean_parser/metrics-yaml.html) files.
+
+Glean itself provides some metrics out of the box.
```
Comment (Dexterp37) [inline docs/appendix/glossary.md]:
```suggestion
Glean itself provides [some metrics out of the box](../user/collected-metrics/metrics.md).
```

File: docs/appendix/glossary.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Ping
+
+A ping is an entity used to bundle related metrics.
+The Glean SDK provides default pings and allows for custom ping, see [Glean Pings](../user/pings/index.md).
+
+## Pipeline
+
+The data pipeline is the infrastructure that collects, stores, and analyzes Telemetry data from our products and logs from various services.
```
Comment (Dexterp37) [inline docs/appendix/glossary.md]:
```suggestion
The data pipeline is the infrastructure that collects, stores, and analyzes telemetry data from our products and logs from various services.
```

File: docs/appendix/glossary.md (hunk 1 above)
Comment (Dexterp37) [inline docs/appendix/glossary.md]:
```suggestion
## Glean Pipeline
```

Comment (Dexterp37) [reply docs/appendix/glossary.md]: Since we're calling the SDK `Glean SDK` we should probably do the same with the pipeline.

Comment (badboy) [reply docs/appendix/glossary.md]: hm, the pipeline does ingest more than just Glean data at the moment though (and the linked article doesn't even mention Glean)

Comment (Dexterp37) [reply docs/appendix/glossary.md]: True, but here we're really talking about the "Glean Pipeline", aren't we? That part of the pipeline has some Glean-specific moving parts (schema generator, transpiler, etc.).

Moreover, this ties into the "Glean" is "Glean SDK" + "Glean Pipeline" + "Glean tools"

Comment (brizental) [reply docs/appendix/glossary.md]: > Moreover, this ties into the "Glean" is "Glean SDK" + "Glean Pipeline" + "Glean tools"

Might be cool to add this statement the the `## Glean` section. But then with a new section about "Glean tools" and what they are.

Comment (Dexterp37) [reply docs/appendix/glossary.md]: Note that that's already mentioned in the [product docs](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).

**Pull request merged**